etsin_finder/frontend/__tests__/combined/qvain/qvain.fileForm.test.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import axios from 'axios'
+import { configure } from 'mobx'
+
+import '../../../locale/translations'
+import { getReferenceData } from '../../__testdata__/referenceData.data'
+import { StoresProvider } from '../../../js/stores/stores'
+import etsinTheme from '../../../js/styles/theme'
+import FileForm from '../../../js/components/qvain/fields/files/ida/forms/fileForm'
+import DirectoryForm from '../../../js/components/qvain/fields/files/ida/forms/directoryForm'
+import { File, Directory, Project } from '../../../js/stores/view/common.files.items'
+import QvainClass from '../../../js/stores/view/qvain'
+import AccessibilityClass from '../../../js/stores/view/accessibility'
+import ElasticQueryClass from '../../../js/stores/view/elasticquery'
+import LocaleClass from '../../../js/stores/view/locale'
+import EnvClass from '../../../js/stores/domain/env'
+import { SaveButton } from '../../../js/components/qvain/general/buttons'
+import { ValidationErrors } from '../../../js/components/qvain/general/errors/validationError'
+
+global.Promise = require('bluebird')
+
+Promise.config({
+  cancellation: true,
+})
+
+const Env = new EnvClass()
```

Comment:
nit: buildStores function from stores/stores could do these automatically.

Reply:
true, this was originally copied from another test file. switched to buildStores